plugin.xml

```diff
@@ -29,7 +29,7 @@
     <repo>https://git-wip-us.apache.org/repos/asf/cordova-plugin-file-transfer.git</repo>
     <issue>https://issues.apache.org/jira/browse/CB/component/12320650</issue>
 
-    <dependency id="cordova-plugin-file" version="^5.0.0" />
+    <dependency id="cordova-plugin-file" version="^6.0.0" />
```

Comment:
@kaynz I think it would be better to make it ">5.0.0".

Reply:
@macdonst Yep, I agree.

Reply:
@kaynz sorry, should be ">=5.0.0" not just ">"

Reply:
Shouldn't we also update `version="1.7.1-dev"`?

Reply:
@macdonst @fikriauliya done